main.cpp

```diff
@@ -23,6 +23,11 @@
 #include "Includes/WirelessPromiscuousDevice.h"
 #include "Includes/XLinkKaiConnection.h"
 
+// Set target to Windows 7 for MSVC
+#if defined(MSVC) && (defined(_WIN32) || defined(_WIN64))
```

Comment:
This should be put into the cmake, not the code

Reply:
Moved to CMakeLists.txt